### DISCUSSION TITLE
Overwriting the account activation / reset password notification and form

### DISCUSSION THREAD

**Discussion by torian257x** [statamic/cms#4598] [Q&A]
hi

we want to use a custom account activation form for users depending on what they are. Lets just say we have vendors and customers.
We have different designs for each and want to show a different activation/reset form for each user.

We have some additional logic as well depending on what the user is (e.g. vendor or customer)
for both we want to set email_verified_at once they activate the account aka once they fill out the account activation form. Not in a separate step.

can we somehow get that functionality as well in the cp e.g.
/admin/cp/users if we choose password reset, we want to send a notification depending on what user it is.

**Comment by jackmcdade** [comment]
There's actually a brand new addon that does **just this**. Check it out! https://statamic.com/addons/jacksleight/members

**Reply by torian257x** [reply]
that is great. But first tag 0.1.0 was yesterday.... is there a way to do it another way as well? any recommendations? we already have a running statamic app, so introducing this brand new addon... not sure how to feel about it honestly. You know what I mean?

**Reply by torian257x** [reply]
maybe I misunderstand the plugin as well... what we want is a custom form for "members" and for "customers" both are 2 different types of members. Does the plugin cover that? Or is it just 1 type of members

**Reply by jackmcdade** [reply]
It should – all "users" are the same, what roles or groups you put them into (and what permissions they get) is up to you there.
